reporter/reporter.go

```diff
@@ -236,6 +245,17 @@ func (t *reporter) reportDerivedMetrics(span tracer.RawSpan) {
 	replaceTag(tags, "application", appName, appFound)
 	replaceTag(tags, "service", serviceName, svcFound)
 
+	redMetricsCustomTags := t.application.Map()
+	if t.redMetricsCustomTagKeys != nil {
+		for _, key := range t.redMetricsCustomTagKeys {
+			if value, found := getAppTag(key, "", span.Tags); found {
+				tags[key] = value
```

Comment:
can redMetricsCustomTagKeys ever contains key that overlaps with previous tag keys?